tests/conformance/common.go

```diff
@@ -160,6 +160,10 @@ func ParseConfigurationMap(t *testing.T, configMap map[string]interface{}) {
 				val = uuid.New().String()
 				t.Logf("Generated UUID %s", val)
 				configMap[k] = val
+			} else if strings.Contains(val, "${{") {
```

Comment:
I wonder if maybe we should change the parsing to use Go templates, which would allow us to put variables also in-between. And also define a function to generate UUIDs.
https://pkg.go.dev/text/template